website/pages/aoc_en.md

```diff
@@ -34,7 +34,10 @@ ended: First place 1500 sek, second place 1100 sek, third place 750 sek, 4:th to
 
 <hr>
 
+<label class="toggle-aoc" for="aoc-trigger"><span class="only-aoc-some">Toggle On</span><span class="only-aoc-all">Toggle Off</span></label>
```

Comment:
If you don't write this on one line, things start to warp around because the spaces are still rendered.